### PR TITLE
XW-57 | add category test for Mercury

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-57

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/mercury/Category.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.wikia.webdriver.elements.mercury;
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/elements/mercury/Category.java]: isn't that wikiamobile skin ? then correct pachage would be elements.wikiamobile

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java [hunk 2]
```diff
@@ -30,8 +30,8 @@ private void init() {
     new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
   }
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java]: we agreed on calling this type of tests "Navigating". Change the class name and group

File: src/test/java/com/wikia/webdriver/testsuites/testSuite.xml [hunk 3]
```diff
@@ -151,6 +151,7 @@
             <class name="com.wikia.webdriver.testcases.mercurytests.old.widgettests.WeiboTests"/>
             <class name="com.wikia.webdriver.testcases.mercurytests.old.widgettests.GoogleFormTests"/>
             <class name="com.wikia.webdriver.testcases.mercurytests.old.RedirectionTests"/>
+            <class name="com.wikia.webdriver.testcases.mercurytests.CategoryTest"/>
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testsuites/testSuite.xml]: The test functionality should be indicated on this level. Form what the test does I can see it's
Navigating (it expects some url change).

**Comment by justnpT** [comment]
Two questions:
1. Why are we creating a wikiamobile test ?
2. This is a multi-application test. How should it be structured then ?

File: src/test/java/com/wikia/webdriver/elements/mercury/Category.java (hunk 1 above)
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/elements/mercury/Category.java]: it is in mercury

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryTest.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
+  }
+
+  @Test(groups = "mercury-category-001")
+  public void category_001_openNavigate() {
+    init();
+
+    category.toggle();
+    category.openLink(0);
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryTest.java]: rename to `openCategoryPage()`

**Pull request merged**